internal/usage/usage.go

```diff
@@ -237,6 +237,7 @@ const (
 	EndpointServiceID                        = "Unique identifier of the private endpoint service for which you want to retrieve a private endpoint."
 	PrivateEndpointIDAzure                   = "Unique identifier of the Azure private endpoint resource."
 	PrivateEndpointIPAddressAzure            = "Private IP address of the private endpoint network interface you created in your Azure VNet."
+	Endpoint                                 = "List of GCP endpoints in the group separated by commas, such as: endpointName1@ipAddress1,...,endpointNameN@ipAddressN"
```

Comment:
[nit] it doesn't have to be separated by commas `--endpoint endpoint-0@10.142.0.2,endpoint-1@10.142.0.3` and `--endpoint endpoint-0@10.142.0.2 --endpoint endpoint-1@10.142.0.3` are both acceptable ways, cc @JuliaMongo